GLM.Rmd

```diff
@@ -0,0 +1,114 @@
+---
+title: "DAS_Group23_GLM"
+output: pdf_document
+date: "2023-03-16"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = F, message = F)
+```
+
+
+# Data Exploration
+## 1
+Create Response variable: Create a new variable indicating whether each item costs more than 1000 Saudi Riyals. Already done in the cleaning part
+
+## 2
+```{r}
+furniture <- read.csv("cleaned_data.csv", stringsAsFactors = T)
+furniture$price <- factor(furniture$price, levels = c(0, 1), labels = c("below", "above"))
+```
+
+
+The relationship category of price.
+```{r}
+library(ggplot2)
+library(tidyverse)
+furniture %>% ggplot(mapping=aes(x=str_trunc(as.character(category), 9, ell="..."), fill=price)) +
+    geom_bar(position="fill") +
+  theme(axis.text.x = element_text(angle = 90, vjust=0.4)) + 
+  xlab("Category") + 
+  ylab("Proportion")
+```
+
+Category of sofas and armchairs has the most proportion of the price above 1000.
+
+Relationship between sellable_online and price.
+```{r}
+furniture %>% ggplot(mapping=aes(x=sellable_online, fill=price)) +
+    geom_bar(position="fill") +
+  theme(axis.text.x = element_text(angle = 90, vjust=0.4)) + 
+  xlab("Sellable online") + 
+  ylab("Proportion")
+```
+
+All unsellable online productions are under 1000
+
+Relationship between other_colors and price.
+```{r}
+furniture %>% ggplot(mapping=aes(x=other_colors, fill=price)) +
+    geom_bar(position="fill") +
+  theme(axis.text.x = element_text(angle = 90, vjust=0.4)) + 
+  xlab("Other colors") + 
+  ylab("Proportion")
+```
+
+The proportion of above 1000 is higher for the other colors.
+
+```{r}
+ggplot(furniture, aes(x = price, y = depth, fill = price)) +
+  geom_boxplot() +
+  labs(x = "Price", y = "Depth")+ 
+  theme(legend.position = "none")
+```
+Depth between 50 and 100 seems like more possible to have price over1000
+
+```{r}
+ggplot(furniture, aes(x = price, y = height, fill = price)) +
+  geom_boxplot() +
+  labs(x = "Price", y = "Height")+ 
+  theme(legend.position = "none")
+```
+
+```{r}
+ggplot(furniture, aes(x = price, y = width, fill = price)) +
+  geom_boxplot() +
+  labs(x = "Price", y = "Width")+ 
+  theme(legend.position = "none")
+```
+
+It Seems like the bigger the furniture is, the higher the price is
+
+# Modeling
+## Model 1
+Build a multiple logistic regression model, use the sellable_one, other_colors, depth, height and width as the predictors, to predict the price.
+
+```{r}
+# Fit a binary logistic regression model
+model1 <- glm(price ~   sellable_online + other_colors + depth + height + width,
```

Comment:
This model will also be useful as part of our variable selection process, so we'll include it in the final .Rmd